pyglet/math.py

```diff
@@ -1437,6 +1449,8 @@ def to_mat4(self) -> Mat4:
         return Mat4(a, b, c, 0.0, e, f, g, 0.0, i, j, k, 0.0, 0.0, 0.0, 0.0, 1.0)
 
     def to_mat3(self) -> Mat3:
+        """Convert the quaternion to a 3 by 3 rotation matrix."""
```

Comment:
This should probably be "Calculate", similar to the `to_mat4` method. "Convert" could be confusing.